app/src/lib/menu-update.ts

```diff
@@ -384,6 +379,30 @@ function getNoRepositoriesBuilder(state: IAppState): MenuStateBuilder {
   return menuStateBuilder
 }
 
+function getRepoIssuesEnabled(state: IAppState): boolean {
```

Comment:
💯 this helper method

are there other helpers in this file that use all of `state`? if that's not an established pattern, then i'd prefer to only pass in `selectedState` or `selectedState.repository` instead of all of `state`, to help keep the scope of this helper constrained.